### DISCUSSION TITLE
Some Good FOSS Recommendations to Add

### DISCUSSION THREAD

**Discussion by ahmednasser293** [ChrisTitusTech/winutil#1909] [Ideas]
Aegisub: https://github.com/Aegisub/Aegisub
Subtitle Editor: https://github.com/SubtitleEdit/subtitleedit
MakeMKV (actually kinda Freemium rather than FOSS but actually legit): https://www.makemkv.com/
System Informer (The Official Successor of Process Hacker): https://github.com/winsiderss/systeminformer
Jackett: https://github.com/Jackett/Jackett
MKVToolNix: https://mkvtoolnix.download/
Inviska MKV Extract (Requires MKVToolNix): https://www.videohelp.com/software/Inviska-MKV-Extract
yt-dlg (awesome GUI for yt-dl and yt-dlp): https://github.com/yt-dlg/yt-dlg
Transmission (Torrent Client): https://github.com/transmission/transmission
MediaInfo: https://github.com/MediaArea/MediaInfo
LockHunter: https://lockhunter.com/
Qttabbar (the best tool to add tabs and some other cool tweaks for Windows 10's File Explorer): https://github.com/indiff/qttabbar

**Comment by DryPringleSoup** [comment]
Nice list! Here a couple I'd like to see added:
avr8js-electron-playground (Arduino and circuit simulator): https://github.com/arcostasi/avr8js-electron-playground
Piped (another good alt client for yt): https://github.com/TeamPiped/Piped
GeoGebra (Math tools app): https://github.com/geogebra/geogebra
_v ( Freemium but not FOSS )_
DaVinci Resolve (video post-production tool): https://www.blackmagicdesign.com/products/davinciresolve/studio
Rectify11 (Win11 tweak tool that fixes a lot of the style inconsistences/weirdness in Win11 & more): https://github.com/Rectify11/Installer
